charts/csi-secret-provider-class/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.dummyDeployment.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{ template "common.metadata" $ }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "csi-secret-provider-class.selectorLabels" . | nindent 6 }}
```

Comment:
There might be something generic in `common` to replace this but I'd need to check first.

Reply:
found nothing. This is why I created `_helpers.tpl` in the first place.

Reply:
i took a look at the original incubator upstream as well and could not find anything there either